code/game/turfs/simulated/floor/light_floor.dm

```diff
@@ -111,3 +111,43 @@
 	var/color_save = color
 	..()
 	color = color_save
+
+// These tiles change color every now and then
+/turf/simulated/floor/light/disco
+	floor_tile = /obj/item/stack/tile/disco_light
+	can_modify_colour = FALSE	// This tile has a set of colours, thus no interfering
+	var/list/available_colors = list("#d41e3c", "#ed7b39", "#fff540", "#77b02a", "#488bd4", "#b0fff1", "#94007a", "#ff417d")	// These are all the colors we can ever become
+	var/current_color	// This is our current color, don't pick it again (integer for faster list accessing)
```

Comment:
```suggestion
	/// This is our current color, don't pick it again (integer for faster list accessing)
	var/current_color	
```

Reply:
Can you elaborate why? All the variables here have a comment on the side, why this one specifically should have a comment on a new line...?